src/validation/validator.ts

```diff
@@ -1,35 +1,24 @@
-import fsPromises from 'fs/promises';
-import { inject, injectable } from 'tsyringe';
-import { Logger } from '@map-colonies/js-logger';
 import Ajv from 'ajv';
 import { JSONSchemaType } from 'ajv';
 import * as ajvKeywords from 'ajv-keywords';
 import betterAjvErrors from 'better-ajv-errors';
-import { SERVICES } from '../common/constants';
+
+const ajv = new Ajv({ $data: true, coerceTypes: true });
+ajvKeywords.default(ajv);
 
 export interface ValidationResponse<T> {
   isValid: boolean;
   errors?: string;
   content?: T;
 }
 
-@injectable()
-export class Validator<T> {
-  private readonly ajv: Ajv;
-  public constructor(@inject(SERVICES.LOGGER) private readonly logger: Logger) {
-    this.ajv = new Ajv({ $data: true });
-    ajvKeywords.default(this.ajv);
+export function validateBySchema<T>(content: unknown, schema: JSONSchemaType<T>): ValidationResponse<T> {
```

Comment:
its more of an ajv wrapper, than validate by schema.

Reply:
done.